src/app/sessions/containers/sessions-search/sessions-search.component.ts

```diff
@@ -153,4 +156,10 @@ export class SessionsSearchComponent implements OnInit {
             hasBackdrop: true
         });
     }
+
+    private filterUnlistedHearingParts(data: HearingPart[]): HearingPart[] {
```

Comment:
I think that we can change the query on the events service, so we just call for unlisted hearing parts instead of filtering here.